content/en/admin/install.md

```diff
@@ -45,7 +46,7 @@ apt install -y \
   g++ libprotobuf-dev protobuf-compiler pkg-config nodejs gcc autoconf \
   bison build-essential libssl-dev libyaml-dev libreadline6-dev \
   zlib1g-dev libncurses5-dev libffi-dev libgdbm-dev \
-  nginx redis-server redis-tools postgresql postgresql-contrib \
+  nginx nodejs redis-server redis-tools postgresql postgresql-contrib \
```

Comment:
I dont like `nodejs` being in the "System packages" section here, and I would prefer `apt update && apt install -y nodejs` to be in the "Node.js" section

Reply:
The only thing is that there is a section for nodejs, and a section for PostgreSQL. Both sections instruct you to install a package repository for the respective software. It seems incongruous that for PostgreSQL you would then install it as part of "system packages" but not nodejs?